pact/coin-contract/coin.pact

```diff
@@ -10,7 +10,7 @@
     [ (defproperty conserves-mass
         (= (column-delta coin-table 'balance) 0.0))
 
-      (defproperty account-structure (account:string)
+      (defproperty account (account:string)
```

Comment:
suggest `valid-account-id`, just `account` is confusing

Reply:
That was an error on my part - i was "account-structure" and i missed a few when calling it something else. Emacs led me astray. I'll go with `valid-account`